doc-build/md051.md

```diff
@@ -38,6 +38,14 @@ attribute can be used to define a fragment:
 An `a` tag can be useful in scenarios where a heading is not appropriate or for
 control over the text of the fragment identifier.
 
+GitHub also supports links to [specific lines][github-lines-links].
```

Comment:
At the end: "... so this rule will not report violations when this syntax is used.".